docs/explanations/curator_data_model.md

```diff
@@ -0,0 +1,389 @@
+
+```json
+{
+  "description": "TBD",
+  "properties": {
+    "Gender": {
+      "description": "TBD",
+      "title": "Gender",
+      "type": "string"
+    },
+    "ID": {
+      "description": "TBD",
+      "title": "ID",
+      "type": "string",
+      "pattern": "[a-f]"
+    }
+  }
+}
+```
+
+### Minimum/Maximum
+
+The range of numeric values this attribute must be in.  The type of this attribute must be "integer", "number", or "integer_list". See [range](https://json-schema.org/understanding-json-schema/reference/numeric#range)  The value of these columns will be appear as the `minimum` and `maximum` of this attribute in the JSON Schema. Both must be numeric values.
+
+Data Model:
+
+| Attribute    | DependsOn                  | columnType  | Minimum | Maximum |
+|---|---|---|---|---|
+| Patient      | "Age, Weight, Expression"  |             |         |         |
+| Age          |                            | integer     | 0       | 120     |
+| Weight       |                            | number      | 0.0     |         |
+| Health Score |                            | number      | 0.0     | 1.0     |
+
+JSON Schema output:
+
+```json
+{
+  "description": "TBD",
+  "properties": {
+    "Age": {
+      "description": "TBD",
+      "title": "Age",
+      "type": "integer",
+      "minimum": 0,
+      "maximum": 120
+    },
+    "Weight": {
+      "description": "TBD",
+      "title": "Weight",
+      "type": "number",
+      "minimum": 0.0
+    },
+    "Health Score": {
+      "description": "TBD",
+      "title": "Health Score",
+      "type": "number",
+      "minimum": 0.0,
+      "maximum": 1.0
+    },
+  }
+}
+```
+
+### Validation Rules (deprecated)
+
+This is a remnant from Schematic. It is still used (for now) to translate certain validation rules to other JSON Schema keywords.
+
+If you are starting a new data model, DO NOT use this column.
+
+If you have an existing data model using any of the following validation rules, follow these instructions to update it:
+
+- `list`: Make sure you are using one of the list-types in the `columnType` column.
+- `regex`: `regex <module> <pattern>`, move the `<pattern>` to the `Pattern` column.
+- `inRange`: `inRange <minimum> <maximum>`, move the `<minimum>` and/or the `<maximum>` to the `Minimum` and `Maximum` columns respectively.
+- `date`: Use the `Format` column with value `date`
+- `url`: Use the `Format` column with value `uri`
+
+## Conditional dependencies
+
+The `DependsOn` and `Valid Values` columns can be used together to flexibly define conditional logic for determining the relevant attributes for a data type.
+
+In this example we have the `Patient` data type. In this case the `Patient` can dbe diagnosed as healthy or with cancer. For Patients with cancer we also want to collect info about their cancer type, and any cancers in their family history.
+
+Data Model:
+
+| Attribute      | DependsOn                      | Valid Values        | Required | columnType  |
```

Comment:
Are all the quotes necessary? Could we use just Diagnosis instead of "Diagnosis"?

Reply:
I think it's needed because this( and Valid Values) are comma seperated lists. If they aren't quoted it can' screw up the parsing when Pandas opens the CSV. If thats so, it's likely you wouldn't need the quotes if it's just one item, but that would need to be tested to be sure.